stylus/utilities/text.styl

```diff
@@ -61,10 +61,71 @@ $midellipsis
         > :first-child
             text-overflow '[...]'
 
+// @stylint off
+fz-tiny =
+    font-size rem(12)!important
+    line-height 1.3!important
+$fz-tiny
+    {fz-tiny}
```

Comment:
We just change `t-left` to `ta-left` for `text-align`, so having `fz` for `font-size` makes sense.

Reply:
Sorry about that, it is obvious to me but you're absolutely right, it is very important to explain the reasons why I chose "fz"  instead of anything else.

I made my choice of syntax based on the very popular et widely used text editor plugin [Emmet.io](https://emmet.io/) by numerous CSS developers because:
- It is actually a common vocabulary for a lot of people;
- It has abbreviations for every CSS properties figured out (I didn't check but I guess if not all then most of it), so it is a reliable reference for naming future utility classes and duplicate free (they really think it through);

I could have used 'fs' but then it would have been difficult to differentiate `font-size` from `font-style`.  Emmet chose `fz` for the size and & `fs` for the style and I figured why reinventing the wheel on that matter, that would be time & brain poorly spent.

Everybody hates naming things when coding. With this, nobody has to do it anymore, for utility classes anyway.

Reply:
I agree with @GoOz. `fz` is like a standard shortcut for `font-size` thanks to Emmet. I'd go for it too.